src/metabase/routes/index.clj

```diff
@@ -53,6 +53,10 @@
     (fn []
       (memoized-load-localization *locale*))))
 
+(defn- load-inlinejs* [resource-name]
```

Comment:
This should be `inline-js`, not `inlinejs`, because it's two words. You wouldn't write `inlineJS`, you'd write `inline JS`